doc/_utils/redirects.py

```diff
@@ -221,12 +221,18 @@
     ("working_with_nrf/nrf70/developing/constrained","device_guides/working_with_nrf/nrf70/developing/constrained"),
     ("ug_nrf70_developing","device_guides/working_with_nrf/nrf70/developing/index"),
     ("working_with_nrf/nrf70/developing/index","device_guides/working_with_nrf/nrf70/developing/index"),
-    ("ug_nrf70_developing_powersave","device_guides/working_with_nrf/nrf70/developing/powersave"),
-    ("working_with_nrf/nrf70/developing/powersave","device_guides/working_with_nrf/nrf70/developing/powersave"),
```

Comment:
@gmarull, these (lines 224 and 225) are old redirections that were defined earlier, should this be retained as is and should we just add the latest redirection on line  226 ("device_guides/working_with_nrf/nrf70/developing/powersave","protocols/wifi/powersave")?

Reply:
It's up to you which set of redirects you want to keep, but if the redirected page is also removed, just update the entry with the current final destination.